### DISCUSSION TITLE
Error: Read ECONNRES

### DISCUSSION THREAD

**Discussion by phxfoto** [requarks/wiki#5417] [Error / Bug Report] [answered]
My install has access to the internet but my language files won't load. I can't get side loading to work either. Here is my boot log. The Read ECONNRES appears to be the error. When I look this up I get information about node.js having an issue; however, I am not sure what needs to change.

-- Logs begin at Mon 2022-02-14 08:24:23 MST, end at Mon 2022-06-20 09:54:18 MST. --
Jun 10 13:03:31 bplinux03 node[6474]: Loading configuration from /var/www/wiki/config.yml... OK
Jun 10 13:03:31 bplinux03 node[6474]: 2022-06-10T20:03:31.614Z [MASTER] info: =======================================
Jun 10 13:03:31 bplinux03 node[6474]: 2022-06-10T20:03:31.617Z [MASTER] info: = Wiki.js 2.5.283 =====================
Jun 10 13:03:31 bplinux03 node[6474]: 2022-06-10T20:03:31.617Z [MASTER] info: =======================================
Jun 10 13:03:31 bplinux03 node[6474]: 2022-06-10T20:03:31.618Z [MASTER] info: Initializing...
Jun 10 13:03:32 bplinux03 node[6474]: 2022-06-10T20:03:32.425Z [MASTER] info: Using database driver pg for postgres [ OK ]
Jun 10 13:03:32 bplinux03 node[6474]: 2022-06-10T20:03:32.434Z [MASTER] info: Connecting to database...
Jun 10 13:03:32 bplinux03 node[6474]: 2022-06-10T20:03:32.535Z [MASTER] info: Database Connection Successful [ OK ]
Jun 10 13:03:33 bplinux03 node[6474]: 2022-06-10T20:03:33.381Z [MASTER] warn: Mail is not setup! Please set the configuration in the administrati>
Jun 10 13:03:33 bplinux03 node[6474]: 2022-06-10T20:03:33.520Z [MASTER] info: Loading GraphQL Schema...
Jun 10 13:03:34 bplinux03 node[6474]: 2022-06-10T20:03:34.559Z [MASTER] info: GraphQL Schema: [ OK ]
Jun 10 13:03:34 bplinux03 node[6474]: 2022-06-10T20:03:34.978Z [MASTER] info: HTTP Server on port: [ 3000 ]
Jun 10 13:03:35 bplinux03 node[6474]: (node:6474) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./public/" in the "exports" fie>
Jun 10 13:03:35 bplinux03 node[6474]: Update this package.json to use a subpath pattern like "./public/*".
Jun 10 13:03:35 bplinux03 node[6474]: (Use `node --trace-deprecation ...` to show where the warning was created)
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.018Z [MASTER] warn: listen EADDRINUSE: address already in use 0.0.0.0:3000
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.518Z [MASTER] info: No new analytics providers found: [ SKIPPED ]
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.565Z [MASTER] info: Loaded 21 authentication strategies: [ OK ]
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.603Z [MASTER] info: No new comment providers found: [ SKIPPED ]
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.621Z [MASTER] info: No new editors found: [ SKIPPED ]
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.660Z [MASTER] info: No new loggers found: [ SKIPPED ]
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.713Z [MASTER] info: No new renderers found: [ SKIPPED ]
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.743Z [MASTER] info: No new search engines found: [ SKIPPED ]
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.784Z [MASTER] info: No new storage targets found: [ SKIPPED ]
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.785Z [MASTER] info: Checking for installed optional extensions...
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.803Z [MASTER] info: Optional extension git is installed. [ OK ]
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.810Z [MASTER] info: Optional extension pandoc was not found on this system. [ SKIPPED ]
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.817Z [MASTER] info: Optional extension puppeteer was not found on this system. [ SKIPPE>
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.818Z [MASTER] info: Optional extension sharp was not found on this system. [ SKIPPED ]
Jun 10 13:03:35 bplinux03 node[6474]: 2022-06-10T20:03:35.824Z [MASTER] info: Authentication Strategy Local: [ OK ]
Jun 10 13:03:36 bplinux03 node[6474]: 2022-06-10T20:03:36.869Z [MASTER] info: (COMMENTS/DEFAULT) Initializing...
Jun 10 13:03:36 bplinux03 node[6474]: 2022-06-10T20:03:36.870Z [MASTER] info: (COMMENTS/DEFAULT) Initialization completed.
Jun 10 13:03:36 bplinux03 node[6474]: 2022-06-10T20:03:36.908Z [MASTER] info: Purging orphaned upload files...
Jun 10 13:03:36 bplinux03 node[6474]: 2022-06-10T20:03:36.917Z [MASTER] info: Syncing locales with Graph endpoint...
Jun 10 13:03:36 bplinux03 node[6474]: 2022-06-10T20:03:36.918Z [MASTER] info: Fetching latest updates from Graph endpoint...
Jun 10 13:03:37 bplinux03 node[6474]: 2022-06-10T20:03:37.156Z [MASTER] info: Purging orphaned upload files: [ COMPLETED ]
Jun 10 13:03:37 bplinux03 node[6474]: 2022-06-10T20:03:37.172Z [MASTER] error: Syncing locales with Graph endpoint: [ FAILED ]
Jun 10 13:03:37 bplinux03 node[6474]: 2022-06-10T20:03:37.173Z [MASTER] error: request to https://graph.requarks.io failed, reason: read ECONNRES>
Jun 10 13:03:37 bplinux03 node[6474]: 2022-06-10T20:03:37.177Z [MASTER] error: Fetching latest updates from Graph endpoint: [ FAILED ]
Jun 10 13:03:37 bplinux03 node[6474]: 2022-06-10T20:03:37.177Z [MASTER] error: request to https://graph.requarks.io failed, reason: read ECONNRES>
Jun 10 13:03:37 bplinux03 node[6499]: Loading configuration from /var/www/wiki/config.yml... OK
Jun 10 13:03:37 bplinux03 node[6499]: 2022-06-10T20:03:37.356Z [JOB] info: Rebuilding page tree...
Jun 10 13:03:38 bplinux03 node[6499]: 2022-06-10T20:03:38.006Z [JOB] info: Using database driver pg for postgres [ OK ]

**Comment by NGPixel** [comment, marked as answer]
You have a much more critical error higher up "warn: listen EADDRINUSE: address already in use 0.0.0.0:3000"

You're trying to start Wiki.js on port 3000 but there's already something listening on that port.

**Reply by phxfoto** [reply]
I have fixed that issue and still no languages will load.
